### ISSUE TITLE
High Oven shows no progress in version 1.1.1.17/ok in 1.1.0.16

### ISSUE THREAD

**Issue by donqixot** [Ephys/TinkersSteelworks#118]
I downloaded latest versions TSteelworks-1.7.10-1.1.1-17.jar and CookieCore-1.7.10-1.4.0-11.jar. I use MultiMC as launcher, created new instance, added forge 10.13.4.1448, added latest TConstruct-1.7.10-1.8.8.jar and Mantle-1.7.10-0.3.2b.jar. I created flat world and constructed high oven 8 blocks high, of which 5 layers are in the ground. High oven controller, one duct and one drain are all on layer 7, which is second layer above ground. They are all on different sides of oven. I click on controller and High Oven GUI is shown, I assume multiblock is ok.

I then put gunpowder, redstone, sand, iron and charcoal and nothing happens, no progress is shown.

I go to mods folder, replace files with previous version TSteelworks-1.7.10-1.1.0-16.jar and CookieCore-1.7.10-1.3.1-8.jar, go back to my world and I see High Oven is smelting iron. I exit, go back to mods folder, replace file with latest version, go back to my world and no progress is shown, even though the state of oven is correct, whatever it was before.

Anything different I should be doing in latest version? I use java 1.8.0_65-b17. I tried to switch forge to latest 1558, also tried on another computer (one Intel, one AMD), same results. Any suggestions?

**Comment by Ephys** [comment]
Hmmm... I can't seem to reproduce, I'll release a new version after this batch of bugfixes. Can you try with that one ?

**Comment by donqixot** [comment]
No, same thing, I tried with version TSteelworks-1.7.10-1.1.2-19 and CookieCore-1.7.10-1.4.0-11. Attached is zip file with my instance from MultiMC, if you place it into your "instances" folder inside MultiMC, you can see for yourself. For some reason, github doesn't take zip file directly, i had to rename it to TSteel.zip.png, just rename it back to zip.


![tsteel zip](https://cloud.githubusercontent.com/assets/12219588/11647162/cc2b9908-9d32-11e5-9b5f-f8902663f673.png)
